lesson7.md

```diff
@@ -0,0 +1,254 @@
+<!-- .slide: id="lesson7" -->
+
+# Basic Frontend - Spring 2020
+
+Lesson 7, Tuesday, 2020-03-24
+
+---
+
+<!-- .slide: id="lesson7:arrays" -->
+
+### Refresher: Types
+
+```js
+// Strings
+let name = "Owen";
+
+// Numbers
+let age = 28;
+
+// Booleans
+let isProgrammer = true;
+```
+
+---
+
+### Arrays
+
+Array is a container type that holds multiple values:
+
+```js
+let cars = ["Saab", "Volvo", "BMW"];
+
+// we create an array using []
+let emptyArray = [];
+
+// we put the values we want in these angled brackets
```

Comment:
Isn't it `square brackets` (https://grammar.collinsdictionary.com/easy-learning/square-brackets) ? Angled seems to be `<>`: https://en.wiktionary.org/wiki/angle_bracket

Reply:
Yes you're right, I think this was copied from the old slide

Reply:
Fixed